### PR TITLE
[IOTDB-4117] fix compact bug ArithmeticException: / by zero

### PR DESCRIPTION
**Pull request by jamber001**
https://issues.apache.org/jira/browse/IOTDB-4117

### CODE REVIEW

**Comment by jamber001** [comment]
@THUMarkLau @choubenson   Could you help review this PR about compact?  ;-)

**Review by THUMarkLau — APPROVED** [review]

**Review by choubenson — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/utils/ReadPointCrossCompactionEstimator.java [hunk 1]
```diff
@@ -93,8 +106,17 @@ private long calculateReadingSeqFiles(List<TsFileResource> seqResources) throws
       // it is max aligned series num of one device when tsfile contains aligned series,
       // else is sub compaction task num.
       int concurrentSeriesNum = fileInfo[2] == -1 ? subCompactionTaskNum : fileInfo[2];
-      long seqFileCost =
-          concurrentSeriesNum * (seqResource.getTsFileSize() * fileInfo[1] / fileInfo[0]);
+      long seqFileCost = 0;
+      if (fileInfo[0] == 0) { // If totalChunkNum ==0, i.e. this unSeq tsFile has no chunk.
```
Comment (choubenson) [inline server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/utils/ReadPointCrossCompactionEstimator.java:110]: Modify the comments as `seq tsFile`

Comment (jamber001) [reply server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/utils/ReadPointCrossCompactionEstimator.java]: Thank your comments.
I have just corrected it.

**Pull request merged**